setup.cfg

```diff
@@ -1,11 +1,32 @@
 [flake8]
-select = E,F,W,N,D
+select = E,F,W,N,D,RST
 ignore = W503,E126,E133,D107
-exclude = .git,__pycache__,sphinx-doc/conf.py,build,dist
+exclude = .git,
+    __pycache__,
+    sphinx-doc/conf.py,
+    run-clang-format.py,
+    build,
+    hoomd/extern,
 max_line_length = 80
 max_doc_length = 80
 hang_closing = True
 docstring-convention = google
+rst-directives =
+    program,
+    option,
+rst-roles =
+    file,
```

Comment:
Is there an order of priority here? If not, I'd sort them alphabetically.

Reply:
This is just the order in which they were added.

Reply:
This config was copied from GSD by the way.

Reply:
I think I'm at least partially responsible for this ordering in GSD, I recall adding a number of these when rewriting some docs. Maybe the GSD CLI? Not sure.